### DISCUSSION TITLE
Przestało działać - nie załadowano (Stopped working - not loaded)

### DISCUSSION THREAD

**Discussion by home409ca** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#485] [General]
Udało się skonfigurować, ale nie działało wybieranie strefy odkurzania. Po pewnym czasie przestało działać. Restart nie pomaga. Mam taki komunikat:

![image](https://user-images.githubusercontent.com/114701129/193428269-356bd32e-abbc-40bb-8f21-0cb1b2aede1c.png)

Nie mam już pomysłu co zrobić. Jak usunąć dodatek, żebym go mógł ponownie zainstalować? Chyba, że problem leży po innej stronie...

Wgrane mam:
[Xiaomi Cloud Map Extractor](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor)
[Xiaomi Vacuum Map card](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-Xiaomi-Vacuum-Map-card)

Czy potrzebne jest też: [Xiaomi Vacuum Map Camera](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Vacuum-Map-Camera) ? Nie bardzo wiem, do czego służy,

------

It managed to configure, but the vacuuming zone selection did not work. After a while it stopped working. Restarting doesn't help. I got this message:

I have no idea what to do anymore. How to remove the add-on so I can reinstall it? Unless the problem is different...

I have uploaded:
[Xiaomi Cloud Map Extractor](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor)
[Xiaomi Vacuum Map card](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-Xiaomi-Vacuum-Map-card)

Is it also needed:  [Xiaomi Vacuum Map Camera](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Vacuum-Map-Camera) ? I don't really know what it's for.

**Comment by PiotrMachowski** [comment]
This error message means that you haven't added this card in dashboard resources.

> Is it also needed: Xiaomi Vacuum Map Camera ? I don't really know what it's for.

No, it's not needed.

**Reply by home409ca** [reply]
Yes and no.
Card already added (correctly), but the message "Not loaded" is still there. Should I be concerned?

**Reply by PiotrMachowski** [reply]
What URL have you used in resources?

**Reply by home409ca** [reply]
`/local/custom_lovelace/xiaomi-vacuum-map-card.js` 
Previously, I created a directory "www" and uploaded the file. 

And why can't I generate room configurations
![image](https://user-images.githubusercontent.com/114701129/193652085-a66161a9-eaee-42f2-856a-c0666d14f3aa.png)
**I already know-parameter "rooms" in file "configuration.yaml"**

BTW. The entity ("camera.xiaomi_cloud_map_extractor") does not have a unique identifier.
![image](https://user-images.githubusercontent.com/114701129/193653253-c15e45cd-c526-484f-8ce7-0870054086e1.png)

**Reply by PiotrMachowski** [reply]
> `/local/custom_lovelace/xiaomi-vacuum-map-card.js`
> Previously, I created a directory "www" and uploaded the file.

So you have installed it both manually and using HACS, I suggest using just HACS instead

> And why can't I generate room configurations

https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317

> BTW. The entity ("camera.xiaomi_cloud_map_extractor") does not have a unique identifier.

Yes, it doesn't have it at this moment

**Reply by home409ca** [reply]
> So you have installed it both manually and using HACS, I suggest using just HACS instead

Yes, because through HACS I had some problems (I do not remember)
I changed the URL to the one from the HACS installation and this time it worked. Strange, but it helped :)


Thanks!
